pixi.toml

```diff
@@ -107,8 +162,33 @@ depends-on = [{ task = "build-tests" }]
 
 [target.linux.tasks.simtest]
 cmd = ["pytest", "$PIXI_PROJECT_ROOT/testing"]
-depends-on = [{ task = "build-tests" }]
 
 [target.linux.tasks.simtest.env]
 NUMBA_CUDA_TEST_BIN_DIR = "$PIXI_PROJECT_ROOT/testing"
 NUMBA_ENABLE_CUDASIM = "1"
+
+[package]
+name = "numba-cuda"
+version = "0.20.0"
```

Comment:
I'd like to automatically compute this, but I'm not sure if that's easily possible with pixi.

Reply:
That's a bit of a shame. We just released `0.20.1` so in the least this needs to be bumped.